test-utils/state-viewer/src/rocksdb_stats.rs

```diff
@@ -68,31 +72,31 @@ impl Data {
     }
 }
 
-fn main() {
-    let home_dir = get_default_home();
+pub fn get_rocksdb_stats(home_dir: &Path, file: Option<PathBuf>) -> anyhow::Result<()> {
     let store_dir = get_store_path(&home_dir);
     let mut cmd = Command::new("sst_dump");
     cmd.arg(format!("--file={}", store_dir.to_str().unwrap()))
         .arg("--show_properties")
         .arg("--command=none"); // For some reason, adding this argument makes execution 20x faster
     eprintln!("Running {:?} ...", cmd);
-    let output = cmd.output().expect("sst_dump command failed to start");
+    let output = cmd.output()?;
+    if !output.status.success() {
+        anyhow::bail!(
+            "failed to run sst_dump, {}, stderr: {}",
+            output.status,
+            std::str::from_utf8(&output.stderr)?
```

Comment:
For diagnostic purposes, you generally want 

https://doc.rust-lang.org/stable/std/string/struct.String.html#method.from_utf8_lossy